app/src/main/java/com/infomaniak/drive/ui/fileShared/FileSharedListFragment.kt

```diff
@@ -122,8 +162,50 @@ class FileSharedListFragment : FileListFragment() {
         )
     }
 
+    private fun downloadAllFiles() {
+        fileSharedViewModel.rootSharedFile.value?.let { file ->
+            drivePermissions?.let { permissions -> requireContext().downloadFile(permissions, file) }
+        }
+    }
+
+    private fun onDriveAndFolderSelected(data: Intent?) {
+        val destinationDriveId = data?.getIntExtra(DESTINATION_DRIVE_ID_KEY, DEFAULT_ID) ?: DEFAULT_ID
+        val destinationFolderId = data?.getIntExtra(DESTINATION_FOLDER_ID_KEY, DEFAULT_ID) ?: DEFAULT_ID
+
+        if (data == null || destinationDriveId == DEFAULT_ID || destinationFolderId == DEFAULT_ID) {
+            showSnackbar(RCore.string.anErrorHasOccurred)
+        } else {
+            fileSharedViewModel.importFilesToDrive(
+                destinationDriveId = destinationDriveId,
+                destinationFolderId = destinationFolderId,
+                fileIds = multiSelectManager.selectedItemsIds.toList(),
+                exceptedFileIds = multiSelectManager.exceptedItemsIds,
+            )
+        }
+    }
+
+    private fun setMainButton(importButton: MaterialButton) {
+        importButton.setOnClickListener {
+            if (AccountUtils.currentDriveId == -1) {
+                showSnackbar(title = "TODO : Show bottomsheet to get app", anchor = importButton)
```

Comment:
Fix this todo

Reply:
I won't dev a full new Ui bottomsheet in this PR because as the back isn't ready yet, we don't even know if this will be kept in the future.
So either we'll remove this button or i'll dev the bottomsheet.
What I can do however is removing the "showSnackbar" to be sure it's never displayed even if we forget about it

Reply:
Add a TODO and redirect to login

Reply:
After discussion, a quick behaviour to wait for the back is to redirect to the login